source/tutorials/facets.toml

```diff
@@ -0,0 +1,4 @@
+
+[[facets]]
+category = "programming_language"
+value = "javascript/typescript"
```

Comment:
Issue:
I think Kafka Connector tutorials are not related to Javascript or Typescript.

Suggestion:
I think the language should be omitted from the Kafka Connector tutorials. There are some Python scripts that use the PyMongo driver and some JSON output, but those aren't used in all of the tutorials.

Reply:
I meant to change this to a genres/tutorials facet 🙃 
Thanks for catching this.